libs/map/src/components/action/layer-action/select-action/multiselect-actions/stories/vl-map-multiselect-actions.stories-arg.ts

```diff
@@ -0,0 +1,13 @@
+import { ArgTypes } from '@storybook/web-components';
+import {
+    mapSelectActionsArgTypes,
+    mapSelectActionsArgs,
+} from '../../select-actions/stories/vl-map-select-actions.stories-arg';
+
+export const mapMultiselectActionsArgs = {
```

Comment:
Kan je hier `: typeof mapSelectActionsArgs` aan toevoegen? Dan krijg je typing in je stories voor deze args.

Reply:
done